### DISCUSSION TITLE
What's the point of having level.Chunk and save.Chunk?

### DISCUSSION THREAD

**Discussion by maxsupermanhd** [Tnze/go-mc#155] [General]
What's the difference and why keep 2 structures that represent same thing?

**Comment by Tnze** [comment]
This is actually a bit like Codec in the official Java code.  The older one (save.Chunk) will be deleted after I fully understand how these things work.

**Reply by maxsupermanhd** [reply]
But how then I will parse chunks from anvil storage?

**Reply by Tnze** [reply]
Should be some way to do that

**Comment by Tnze** [comment]
This is actually a bit like Codec in the official Java code.  The older one (save.Chunk) will be deleted after I fully understand how these things work.
